apps/site/next-data/generators/supportersData.mjs

```diff
@@ -1,28 +1,24 @@
+import { OPENCOLLECTIVE_MEMBERS_URL } from '#site/next.constants.mjs';
+
 /**
  * Fetches supporters data from Open Collective API, filters active backers,
  * and maps it to the Supporters type.
  *
- * @returns {Promise<Array<import('#site/types/supporters')>>} Array of supporters
+ * @returns {Promise<Array<import('#site/types/supporters').OpenCollectiveSupporter>>} Array of supporters
  */
-async function fetchOpenCollectiveData() {
-  const endpoint = 'https://opencollective.com/nodejs/members/all.json';
-
-  const response = await fetch(endpoint);
-
-  const payload = await response.json();
-
-  const members = payload
-    .filter(({ role, isActive }) => role === 'BACKER' && isActive)
-    .sort((a, b) => b.totalAmountDonated - a.totalAmountDonated)
-    .map(({ name, website, image, profile }) => ({
-      name,
-      image,
-      url: website,
-      profile,
-      source: 'opencollective',
-    }));
-
-  return members;
-}
-
-export default fetchOpenCollectiveData;
+export default () =>
+  fetch(OPENCOLLECTIVE_MEMBERS_URL)
+    .then(response => response.json())
+    .then(payload =>
+      payload
+        .filter(({ role, isActive }) => role === 'BACKER' && isActive)
+        .sort((a, b) => b.totalAmountDonated - a.totalAmountDonated)
+        .map(({ name, website, image, profile }) => ({
+          name,
+          image,
+          url: website,
+          profile,
+          source: 'opencollective',
+        }))
+    )
+    .catch(() => []);
```

Comment:
I don't know if silently ignore the error is a good idea

Reply:
What was the initial problem? I'm not sure I understood it

Reply:
Yes, absolutely, it should crash, otherwise if this fails... Well, no data.

We could add retry-logic tho